hibernate-core/src/test/java/org/hibernate/orm/test/query/hql/StandardFunctionTests.java

```diff
@@ -888,6 +896,7 @@ public void testAggregateFunctions(SessionFactoryScope scope) {
 
 	@Test
 	@SkipForDialect(dialectClass = DerbyDialect.class, reason = "Derby doesn't support formatting temporal types to strings")
+	@SkipForDialect(dialectClass = FirebirdDialect.class, reason = "Firebird doesn't support formatting temporal types to strings")
```

Comment:
Is there no format function? I mean we could emulate this by using the extraction functions since we require the pattern to be a literal anyway, but I'd rather use a builtin format function if there is one.

Reply:
Unfortunately, no: there is http://tracker.firebirdsql.org/browse/CORE-1314 (which is a bit too generic to my taste), and I have now created http://tracker.firebirdsql.org/browse/CORE-6507

Reply:
It's up to you if you want to support this by emulation through extraction functions. I'm fine with ignoring this test.